presto-main/src/main/java/com/facebook/presto/connector/system/QuerySystemTable.java

```diff
@@ -108,6 +118,17 @@ public RecordCursor cursor(ConnectorTransactionHandle transactionHandle, Connect
         return table.build().cursor();
     }
 
+    private static Block toBlock(ResourceGroupId resourceGroupId)
```

Comment:
toBlock sounds pretty generic - can you make this function name more descriptive? e.g. `resourceGroupIdToBlock`